templates/states.yml

```
@@ -9,48 +9,48 @@
 
   - id: applied
     label: Applied
-    aliases: [aplicado, enviada, aplicada, sent]
+    aliases: [ 'aplicado', 'enviada', 'aplicada', 'sent', 'başvuruldu', 'basvuruldu' ]
     description: Application submitted
     dashboard_group: applied
 
   - id: responded
     label: Responded
-    aliases: [respondido]
+    aliases: [ 'respondido', 'yanıt verildi', 'yanit verildi' ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)